checkoutmanager/dirinfo.py

```diff
@@ -259,6 +324,22 @@ class GitDirInfo(DirInfo):
     vcs = 'git'
 
     @capture_stdout
+    def cmd_rev(self):
+        print(self.directory)
+        os.chdir(self.directory)
+        print(system("git show -q"))
+
+    @capture_stdout
+    def cmd_cu(self):
+        output = system("git pull --dry-run")
```

Comment:
Maybe `git fetch`? That already gets the code, without updating the working copy.